.github/workflows/rc.yml

```diff
@@ -168,7 +168,7 @@ jobs:
       fail-fast: false
       matrix:
         platform:
-          - { runs_on: macos-13, arch: "x86_64"}
+          - { runs_on: macos-latest, arch: "aarch_64"}
```

Comment:
We can remove this pattern entirely.

We just drop support for Intel macOS.

Reply:
i made the change to replace macos-13 with macos-15-intel per https://docs.github.com/en/actions/reference/runners/github-hosted-runners#standard-github-hosted-runners-for-public-repositories